parlai/scripts/torchscript.py

```diff
@@ -34,9 +32,12 @@ def export_model(opt: Opt):
         from parlai.torchscript.modules import TorchScriptGreedySearch
 
     overrides = {
-        "no_cuda": True,  # TorchScripting is CPU only
-        "model_parallel": False,  # model_parallel is not currently supported when TorchScripting
+        "model_parallel": False,  # model_parallel is not currently supported when TorchScripting,
     }
+    if opt.get("script_for_gpu", False):
```

Comment:
Hmm instead of adding the `--script-for-gpu` flag, could we just reuse the value of the `no_cuda` key?